test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/OpenIdConnectEventTests.cs

```diff
@@ -62,6 +64,7 @@ public class OpenIdConnectEventTests
                 OnRedirectToIdentityProvider = RedirectNotImpl,
                 OnRedirectToIdentityProviderForSignOut = RedirectNotImpl,
                 OnRemoteSignOut = RemoteSignOutNotImpl,
+                OnSignedOutCallbackRedirect = RedirectToSignedOutNotImpl,
```

Comment:
The fact you have to do that every time you add a new event makes me think this syntax is not the best one.

You could muchly simplify the OIDC and JWT tests if you use a delegate-based syntax:

```csharp
var server = CreateServer(options =>
{
    options.Events.OnMessageReceived = context => { ... };

    // Don't set the other events as a default "throw a NotImplementedException"
    // event handler is already registered by the CreateServer() test helper.
});
```